graphstorm-processing/graphstorm_processing/data_transformations/dist_transformations/dist_hf_transformation.py

```diff
@@ -0,0 +1,123 @@
+"""
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License").
+You may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from typing import Sequence
+from pyspark.sql import DataFrame
+from pyspark.sql.types import ArrayType, IntegerType, StructType, StructField
+from pyspark.sql.functions import udf
+
+import numpy as np
+from transformers import AutoTokenizer
+
+from graphstorm_processing.constants import HUGGINGFACE_TOKENIZE
+from .base_dist_transformation import DistributedTransformation
+
+
+def apply_norm(
```

Comment:
As with the comment for the feature config, why do we call this a norm? That has a very specific meaning in the [context of numerical data](https://en.wikipedia.org/wiki/Norm_(mathematics)) which doesn't apply here.